playbooks/libwww-prod.yml

```diff
@@ -6,6 +6,7 @@
     - run_not_in_container: true
     - drupal_version_branch: develop
```

Comment:
running on development branch?

Reply:
@kayiwa it is not really used now that capistrano is deploying the code, but it is a fair point.  I changed the commit to remove the reference.